internal/third_party/kubernetes/storage/util.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Looks like this is just used in the `tests/framework/utils` package, so this could move there.

Reply:
Done.